src/rpc/introduction.md

```diff
@@ -0,0 +1,39 @@
+# Remote Procedure Calls (RPC)
+
+Communication between services occurs over remote procedure calls. RPCs happen all the time in distributed systesm. You browser had to make at least one RPC to this website to obtain this webpage.
+
+TiKV, as a distributed system involving a number of nodes, uses RPCs to communicate between each node, as well as the placement driver and clients.
+
+As it turns out, exposing functionality as a remote interface isn't a trivial. Networks are unreliable, systems are diverse and evolving, a huge variety of languages and formats exist, and even things like encoding are hard!
+
+## On the shoulders of giants
+
+Over the past decades the field of computing has largely settled on a few common standards.

```

Comment:
Unfortunately, I think this is overstating it - there are a lot of ways to do IPC/network communication and it doesn't seem like we've settled on much above the network protocols